coding-concepts.js

```diff
@@ -8,36 +8,38 @@
 // --------------------1) What will this log?
 
 const cohort = "Hotel 2022"
-// console.log(cohort.split(""))
+//  console.log(cohort.split(""))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: "H","0","t","e","l", , "2","0","2","2"
+// b) Verify and explain: H', 'o', 't', 'e', 'l', ' ', '2', '0','2', '2', 
+// i was almost right, the result came back with apostrophe and not quotation. the .split() built in method takes what is in a array and splits it into individual strings. 
```

Comment:
I wouldn't get tripped up on which strings came back in the return but you got the concept of .split("") absolutely correct. It is split up per letter into their own strings. String syntax is recognizable as single or double quotes, and the special case of string interpolation. Good job Ney!